media/css/mozorg/contribute/friends.less

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
```

Comment:
In this instance, I value consistency. @alexgibson has a good point about the mental shift required when moving to updating pages with a different box model approach.

My thoughts are to apply `border-box` on specific elements when/where needed until the time we shift *all* bedrock pages to use it by default.

FWIW, we ran in to a problem with this approach when applying the Fx Family nav to `/firefox/os/` (the widget case mentioned above).